src/System.Text.Json/src/System/Text/Json/Document/JsonProperty.cs

```diff
@@ -99,5 +101,8 @@ public override string ToString()
         {
             return Value.GetPropertyRawText();
         }
+
+        private string DebuggerDisplay
+            => Value.Type == JsonValueType.Undefined ? "<Undefined>" : $"\"{ToString()}\"";
```

Comment:
I think this should be `name: value` pair.

Reply:
We said `ToString()` was useful for debugging in the comments, which is why I chose it.

Reply:
`ToString()` is fine for the `value`. But, we may want to append the `name` as well.

Reply:
Yep, and that's why `ToString()` already has it.  (PropertyRawText is from the open quote of the property to the end of the value token(set))

Reply:
> (PropertyRawText is from the open quote of the property to the end of the value token(set))

Perfect :)